src/components/ChannelPreview/ChannelPreviewMessenger.js

```diff
@@ -151,11 +151,10 @@ class ChannelPreviewMessenger extends PureComponent {
           <Message
             unread={this.props.unread > 0 ? this.props.unread : undefined}
           >
-            {!this.props.latestMessage
-              ? t('Nothing yet...')
```

Comment:
so we're getting rid of this placeholder message?

Reply:
Its kind of redundant!! We already set it in ChannelPreview component - https://github.com/GetStream/stream-chat-react-native/blob/833ed4784c0a700194f2874089baa02af6734ee6/src/components/ChannelPreview/ChannelPreview.js#L67